cmd/nebula-cert/sign.go

```diff
@@ -404,6 +404,14 @@ func newKeypair(curve cert.Curve) ([]byte, []byte) {
 	}
 }
 
+func x25519KeyFromPriv(priv []byte) ([]byte, []byte) {
+	pubkey, err := curve25519.X25519(priv, curve25519.Basepoint)
+	if err != nil {
+		panic(err)
```

Comment:
In most cases, I prefer to bubble the error back up and let the caller decide to panic